fscollections/views.py

```diff
@@ -0,0 +1,279 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.contrib import messages
+from django.db import transaction
+from django.http import Http404, HttpResponse, HttpResponseRedirect, JsonResponse
+from django.shortcuts import get_object_or_404, render
+from django.urls import reverse
+
+from fscollections.models import *
+from fscollections.forms import *
+from sounds.models import Sound
+from sounds.views import add_sounds_modal_helper
+from utils.pagination import paginate
+from utils.downloads import download_sounds
+
+
+@login_required
+def collection(request, collection_id):
+    user = request.user
+    is_owner = False
+    is_maintainer = False
+    maintainers = []
+    # if no collection id is provided for this URL, render the oldest collection
+    # only show the collections for which you're the user(owner)
+    if not collection_id:
+        # this could probably be a JsonResponse message indicating an error
+        return HttpResponseRedirect(reverse('your-collections'))
```

Comment:
Why this redirect? I think we should make `collection_id` mandatory and that's it. In fact, isn't that already mandatory? I think this `if` is not needed?